deploy-with-init-params.md

```diff
@@ -20,51 +60,72 @@ Do not forget to stop it once you are done developing
 forgae node --stop
 ```
 
-## Step 1. Update your example project
-Lets add some state and init parameters to our example contract.
+## Step 1: Update your ExampleContract.aes
+
+Lets add some state and init parameters to our example contract which can be found at **contracts/ExampleContract.aes**
+
 ```
 contract ExampleContract =
-  record state = 
-    { savedNumber : int }
 
-  public stateful function init(num : int) = 
-    { savedNumber = num }
+  record state = { savedString : string }
+
+  function init() = { savedString = "aeternity" }
+
+  public function getString() : string =
+	  state.savedString
 
-  public function savedNumber() : int = state.savedNumber
+  public stateful function registerString(word : string) =
+	  put(state { savedString = word})
 ```
 
-As you can see the contract now has a state variable `savedNumber` of type int. The initial value will be passed by the init function. We've also added a read function for this value.
+As you can see the contract now has a state variable `savedString` of type `string`. The initial value will be passed by the init function. We've also added a read function `getString` for this value and a write function `registerString` to write and change the value in `savedString`.
 
 Run forgae compile to verify that your contract compiles successfully
 
 ```
 forgae compile
 ```
 
-## Step 2. Change our default deploy script
-Lets change our default deploy script to add parameters. The parameters of the init functions are always passed as tuple. Here is how our new deploy script deploy function looks like
+## Step 2. Update your deploy.js
+
+Lets add some parameters to our example deploy script which can be found at **deployment/deploy.js**. The parameters of the init functions are always passed as tuple. Here is how our new deploy script looks like
 
 ```
-const deploy = async (network, privateKey) => {
-	let deployer = new Deployer(network, privateKey)
+const Deployer = require('forgae-lib').Deployer;
+
+const deploy = async (network, privateKey, compiler) => {
+    let deployer = new Deployer(network, privateKey, compiler)
+
+    let contract = await deployer.deploy("./contracts/ExampleContract.aes")
 
-	let contract = await deployer.deploy("./contracts/ExampleContract.aes", gasLimit, `(42)`)
+    // Getting savedString value in our ExampleContract
+    let getString = await contract.callStatic('getString')
```

Comment:
from sdk 4.1.0 :"ACI: Automatically decide to send transaction on-chai or call-static." and `callStatic` no longer exists just use `call`, if I'm not wrong.